pkg/tfvars/tfvars.go

```diff
@@ -92,12 +92,9 @@ func TFVars(cfg *types.InstallConfig, bootstrapIgn, masterIgn, workerIgn string)
 		}
 
 		config.AWS = aws.AWS{
-			Endpoints: aws.EndpointsAll, // Default value for endpoints.
-			Region:    cfg.Platform.AWS.Region,
-			ExtraTags: cfg.Platform.AWS.UserTags,
-			External: aws.External{
```

Comment:
This is the only code that could touch the `External` type, so we should drop that type and continue pruning any other orphans in that direction.